### DISCUSSION TITLE
cuts with madanalysis5

### DISCUSSION THREAD

**Discussion by fotonfoton123** [MadAnalysis/madanalysis5#102] [General]
Dear Authors,

I have a question related to cuts with madanalysis5. I examine the process pp>aaa with default cuts. I want to plot photons in the final state with madanalysis5. Madanalysis5 can be drawn plots related to pt[a]-Events, pt[a[1]]-Events, pt[a[2]]-Events, pt[a[3]]-Events. What exactly is the difference between these plots?

Best Regards.

**Comment by jackaraz** [comment]
Hi @fotonfoton123, `a[1]` stands for the leading photon, `a[2]` second leading photon and `a[3]` third leading photon for $p_T$ ordered collection (by default, but the ordering can be changed).

**Reply by fotonfoton123** [reply]
Thank you for your response. What does the plot related to pt[a]-Events tell?

**Reply by jackaraz** [reply]
That refers to the scalar sum of all $p_T$'s in your photon collection.

**Reply by fotonfoton123** [reply]
Thank you for your response.
Best Regards.
